include/extractor/edge_based_graph_factory.hpp

```diff
@@ -42,6 +42,33 @@ namespace osrm
 namespace extractor
 {
 
+namespace lookup
+{
+// Set to 1 byte alignment
+#pragma pack(push, r1, 1)
```

Comment:
Yeah sure, but I can't seem to find the `pack(push, r1, 1)` syntax:
https://gcc.gnu.org/onlinedocs/gcc-6.1.0/gcc/Structure-Layout-Pragmas.html

I guess it's just a named stack to push / pop values on / off (?). Maybe just do

    struct __attribute__ ((packed))

on the one struct where it is needed and `static_assert` on all of the struct's sizes.

(Unrelated: it seems like GCC 6+ supports a pragma for endianess. Interesting.)